src/tests/nested-containers.test.ts

```diff
@@ -0,0 +1,650 @@
+import fetch from 'node-fetch';
+const { TextEncoder, TextDecoder } = require('util');
+
+// Code for gathering coverage for src/ not dist/, otherwise tests should test the built files in dist/
+let eosjs;
+let eosjs_jssig;
+if (process.env.COVERAGE_TESTING === 'true') {
+    eosjs = require('../');
+    eosjs_jssig = require('../eosjs-jssig');
+} else {
+    eosjs = require('../../dist');
+    eosjs_jssig = require('../../dist/eosjs-jssig');
+}
+
+const { JsonRpc, Api } = eosjs;
+const { JsSignatureProvider } = eosjs_jssig;
+
+const privateKey = '5JuH9fCXmU3xbj8nRmhPZaVrxxXrdPaRmZLW1cznNTmTQR2Kg5Z';
+
+const rpc = new JsonRpc('http://localhost:8888', { fetch });
+const signatureProvider = new JsSignatureProvider([privateKey]);
+const api = new Api({ rpc, signatureProvider, textDecoder: new TextDecoder(), textEncoder: new TextEncoder() });
+
+describe('eosjs-nested-container', () => {
+
+    it('should test Multi Index nested containers', async () => {
+        jest.setTimeout(30000);
+
+        await api.getAbi('nestcontnmi');
```

Comment:
Can you add a comment here to specify the contract account nestcontnmi is from launching eos/tests/nested_container_multi_index_test.py and also specify the related nested_container_multi_index.cpp contract can be found from eos/unittests/test-contracts/nested_container_multi_index ?